controllers/drplacementcontrol_controller.go

```diff
@@ -257,16 +285,105 @@ func (r *DRPlacementControlReconciler) Reconcile(ctx context.Context, req ctrl.R
 
 	if d.mcvRequestInProgress {
 		duration := d.getRequeueDuration()
-		logger.Info(fmt.Sprintf("Requeing after %v", duration))
+		r.Log.Info(fmt.Sprintf("Requeing after %v", duration))
 
 		return reconcile.Result{RequeueAfter: duration}, nil
 	}
 
 	return ctrl.Result{Requeue: true}, nil
 }
 
+func (r *DRPlacementControlReconciler) getDRPolicy(ctx context.Context,
+	name, namespace string) (*rmn.DRPolicy, error) {
+	drPolicy := &rmn.DRPolicy{}
+
+	err := r.Client.Get(ctx, types.NamespacedName{Name: name, Namespace: namespace}, drPolicy)
+	if err != nil {
```

Comment:
(nit) Would logging the failure help in debugging?

Reply:
sure